.github/workflows/pr-ci-ok-only.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow is a version of `on-pr.yml` that runs in exactly the cases that `on-pr.yml` doesn't. That is, where
+# `on-pr.yml` skips execution due to `paths-ignore`, this workflow runs only in those cases. We do this so that we can
+# always produce a `ci-ok` check, which we configure as a required passing step on all PRs to be merged into the
+# repository.
+#
+# This is an officially blessed pattern for handling skipped but required status checks. See
+# https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks
+# for more information.
+
+on:
+  pull_request:
+    paths-ignore:
+      # Skip all files *except CHANGELOG* changes.
+      - '**'
+      - 'CHANGELOG.md'
+      - 'CHANGELOG_PENDING.md'
+  push:
+    branches:
+      - main
+    tags:
+      - sdk/v*.*.*
+    paths-ignore:
+      # Skip all files *except CHANGELOG* changes.
+      - 'CHANGELOG.md'
+      - 'CHANGELOG_PENDING.md'
+  repository_dispatch:
+    types:
+      - run-acceptance-tests-command
```

Comment:
I don't think we need this workflow in these cases.  It doesn't matter for pushes to `main`, but I think the `repository_dispatch` bit is wrong, this shouldn't run when `/run-acceptance-tests` is run I think (do we even have that in this repo?)

Reply:
Happy to give it a shot -- removed.